game.py

```diff
@@ -20,7 +20,29 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    wrong_guesses_count = 0
+    wrong_guesses_list = []
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    
+    while wrong_guesses_count < SNOWMAN_MAX_WRONG_GUESSES:
+        print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        print_snowman_graphic(wrong_guesses_count)
+        
+        letter_guess = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        
+        if letter_guess in snowman_word:
+            for letter in snowman_word:
+                if letter == letter_guess:
+                    correct_letter_guess_statuses[letter] = True
+            if is_word_guessed(snowman_word, correct_letter_guess_statuses):
+                print("Congratulations, you win!")
+                return
```

Comment:
Nice use of the `return` keyword. This allows for us to exit not only the while loop early if this condition is met but also the function as a whole.